autopush_rs/src/server/mod.rs

```diff
@@ -338,9 +342,19 @@ impl Server {
         } else {
             ServiceChangeTracker::new(Vec::new())
         };
+        let region = if env::var("AWS_PROFILE").unwrap_or("default".to_string()) == "test" {
+            Region::Custom {
+                endpoint: "http://localhost:8000".to_string(),
+                name: "local_dynamodb".to_string(),
+            }
+        } else {
+            Region::UsEast1
```

Comment:
don't hardcode, you want Region::default() so it's read from the AWS_DEFAULT_REGION env var